VirtoCommerce.Platform.Web/Modularity/ManifestModuleCatalog.cs

```diff
@@ -170,25 +170,50 @@ public override void Validate()
             return result;
         }
 
+        private static void CopyLocalizationAssemblies(string sourceDirectoryPath, string targetDirectoryPath)
+        {
+            var cultureNames = ConfigurationHelper.SplitAppSettingsStringValue("VirtoCommerce:Resources:CultureNames");
+
+            foreach (var languageCode in cultureNames)
+            {
+                if (!string.IsNullOrEmpty(languageCode))
+                {
+                    var separator = Path.DirectorySeparatorChar;
+                    var localizationDirectoryPath = Path.Combine(sourceDirectoryPath, $"{languageCode}{separator}");
+                    var localizationTargetDirectoryPath = Path.Combine(targetDirectoryPath, $"{languageCode}{separator}");
```

Comment:
maybe names "directoryPath" and "targetDirectoryPath" be enough for these variables?

Reply:
The algorithm had been changed. Names had been simplified a bit.